templates/buildkite-agent-hook-environment.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+[[ -f "$HOME/.bash_profile" ]] && source "$HOME/.bash_profile"
```

Comment:
Will now work even if the file is removed - and only source if the file exists.

Reply:
Cool.

Please surround environment variable in braces to prevent spaces in the value breaking up the test or command?